.github/CHANGELOG.md

```diff
@@ -9,23 +9,47 @@
   in serial by calling the `execute()` method.
   [(#840)](https://github.com/PennyLaneAI/pennylane/pull/840)
 
+* The `QuantumTape` class now contains basic resource estimation functionality. The method
+  `tape.get_resources()` returns a dictionary with a list of the constituent operations and the
+  number of times they were run. Similarly, `tape.get_depth()` computes the circuit depth.
+  [(#862)](https://github.com/PennyLaneAI/pennylane/pull/862)
+  
 * Adds the square root X gate `SX`.
-  [(#871)](https://github.com/PennyLaneAI/pennylane/pull/871)
-
-  ```python
-  dev = qml.device("default.qubit", wires=1)
+[(#871)](https://github.com/PennyLaneAI/pennylane/pull/871)
 
-  @qml.qnode(dev)
-  def circuit():
+    ```python
+    dev = qml.device("default.qubit", wires=1)
+    
+    @qml.qnode(dev)
+    def circuit():
       qml.SX(wires=[0])
       return qml.expval(qml.PauliZ(wires=[0]))
-  ```
+    ```
```

Comment:
The whitespace was correct here, might need a revert (e.g., the link `  [(#871)](https://github.com/PennyLaneAI/pennylane/pull/871)` was correct with the leading whitespace, also the indentation for the code snippet)